src/platform.rs

```diff
@@ -97,6 +108,9 @@ impl Platform {
             Platform::AVX512 => 16,
             #[cfg(blake3_neon)]
             Platform::NEON => 4,
+            #[cfg(blake3_wasm32_simd)]
+            // TODO is it 8 or 4??? SSE4 has 4...
```

Comment:
My understanding is that Wasm SIMD always uses 128-bit vectors, which would imply degree 4 (128/32). Am I missing anything?

Reply:
Exactly, but I see that for SSE4 that has same vector size, `pub const MAX_SIMD_DEGREE: usize = 8;` is defined.